lang/codegen/src/generator/common.rs

```diff
@@ -240,7 +240,7 @@ pub(crate) mod casper {
             #[cfg(any(feature = "casper", feature = "casper-livenet"))]
             impl odra::casper::casper_types::bytesrepr::FromBytes for #struct_ident {
                 fn from_bytes(bytes: &[u8]) -> Result<(Self, &[u8]), odra::casper::casper_types::bytesrepr::Error> {
-                    let (_, bytes): (String, _) = odra::casper::casper_types::bytesrepr::FromBytes::from_bytes(bytes)?;
+                    let (_, bytes): (alloc::string::String, _) = odra::casper::casper_types::bytesrepr::FromBytes::from_bytes(bytes)?;
```

Comment:
Shouldn't we reexport `alloc` as `odra::reexport::alloc`?

Reply:
Added the `prelude` mod that gives a  stable interface between `std` and `no_std` env, then we write
```rust
use odra::prelude::vec::Vec;
use odra::prelude::string::String;
```